pkg/namesgenerator/names-generator.go

```diff
@@ -475,6 +475,9 @@ var (
 		// Mary Lou Jepsen, was the founder and chief technology officer of One Laptop Per Child (OLPC), and the founder of Pixel Qi. https://en.wikipedia.org/wiki/Mary_Lou_Jepsen
 		"jepsen",
 
+		// Steven Paul Jobs (February 24, 1955 – October 5, 2011) was an American entrepreneur, industrial designer, business magnate, media proprietor, and investor. https://en.wikipedia.org/wiki/Steve_Jobs
+		"jobs",
```

Comment:
Sorry, the list has been freezed

https://github.com/moby/moby/blob/4f3c5b2568c28a4e0fd1b69ec6f2e0a0715d8cf5/pkg/namesgenerator/names-generator.go#L1-L11